### PR TITLE
Added allow_auto_merge_when_builds_pass

### PR DESCRIPTION
**Pull request by variableresistor**
Atlassian added a new option to auto-merge when all other merge conditions are fulfilled.

### CODE REVIEW

**Review by eelkram — CHANGES_REQUESTED** [review]
Thanks for the update, just need to version bump the .psd1 file as well so the module can publish.

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,9 @@ _These will be removed in the next major release_
 
 - N/A
 
+## 0.32.0
```
Comment (variableresistor) [inline CHANGELOG.md:9]: @eelkram the version bump here isn't correct?

Comment (eelkram) [reply CHANGELOG.md:9]: This is fine, but you also have to bump the version in the .psd1 file as that is what is used to publish the module to the PSGallery.

Comment (eelkram) [reply CHANGELOG.md:9]: I think you skipped past step 1 in https://github.com/beyondcomputing-org/Atlassian.Bitbucket/blob/master/CONTRIBUTING.md

Comment (eelkram) [reply CHANGELOG.md:9]: You need to make the same version change here: https://github.com/beyondcomputing-org/Atlassian.Bitbucket/blob/master/Atlassian.Bitbucket.psd1#L15

Comment (variableresistor) [reply CHANGELOG.md:9]: Oops... I knew that. :-)

**Review by eelkram — APPROVED** [review]
Thanks, looks good.

**Pull request merged**